### PR TITLE
Many Things Lib + Moving Platforms

### PR DESCRIPTION
**Pull request by jumpsplat120**

### CODE REVIEW

File: utilities/bin/core/commands.dsc [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+    usage: <proc[lib_command_usage].context[lib_generic_data|command.usage.denchant]>
+    aliases:
+        - dchant
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.denchant|<player.if_null[null]>].if_null[false]>
+    tab completions:
+        1: <server.enchantment_keys>
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc]: This tag is deprecated in favor of [`<server.enchantments.parse[name]>`](https://meta.denizenscript.com/Docs/Search/server.enchantments)

```suggestion
        1: <server.enchantments.parse[name]>
```

File: utilities/bin/core/commands.dsc [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[null]>].if_null[true]>
+    script:
+        - if <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[server_or_command_block]>].if_null[true]>:
+            - if <player.exists>:
+                - if <context.args.size> == 0:
+                    - define warn <script[lib_config].parsed_key[color.warning]>
+                    - define soft <script[lib_config].parsed_key[color.soft_server_notice]>
+                    - if <queue.list.size> > 1:
+                        - foreach <queue.list.exclude[<queue>]> as:queue:
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc]: These two tags are deprecated in favor of [<util.queues>](https://meta.denizenscript.com/Docs/Search/util.queue)
```suggestion
                    - if <util.queues.size> > 1:
                        - foreach <util.queues.exclude[<queue>]> as:queue:
```

File: utilities/bin/core/commands.dsc (hunk 1 above)
Comment (jumpsplat120) [reply utilities/bin/core/commands.dsc]: Probably, I 'm not keeping up with all of monkey's updates, especially for stuff I don't use constantly.

File: utilities/bin/core/commands.dsc (hunk 2 above)
Comment (jumpsplat120) [reply utilities/bin/core/commands.dsc]: Yes, this is also true.

File: utilities/bin/core/commands.dsc [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+    usage: <proc[lib_command_usage].context[lib_generic_data|command.usage.denchant]>
+    aliases:
+        - dchant
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.denchant|<player.if_null[null]>].if_null[false]>
+    tab completions:
+        1: <server.enchantment_keys>
+        2: <proc[lib_numeric_list].context[<server.enchantment_start_level[<context.args.first>]>|<server.enchantment_max_level[<context.args.first>]>]>
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc:459]: with the new enchantment update, the `server.enchantment_start_level` was deprecated in favor of [`enchantmenttag.min_cost[<level>]`](https://meta.denizenscript.com/Docs/Search/enchantmenttag.min_cost)

File: utilities/bin/core/commands.dsc [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+    usage: <proc[lib_command_usage].context[lib_generic_data|command.usage.denchant]>
+    aliases:
+        - dchant
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.denchant|<player.if_null[null]>].if_null[false]>
+    tab completions:
+        1: <server.enchantments.parse[name]>
+        2: <proc[lib_numeric_list].context[<server.enchantment_start_level[<context.args.first>]>|<server.enchantment_max_level[<context.args.first>]>]>
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc]: `server.enchantment_start_level[<enchantment_name>]` is deprecated in favor of [`enchantmenttag.min_level`](https://meta.denizenscript.com/Docs/Search/enchantmenttag.min_level)

`server.enchantment_max_level[<enchantment_name>]` is deprecated in favor of [`enchantmenttag.max_level`](https://meta.denizenscript.com/Docs/Search/enchantmenttag.max_level)


```suggestion
        2: <proc[lib_numeric_list].context[<enchantment[<context.args.first>].min_level>|<enchantment[<context.args.first>].max_level>]>
```

File: utilities/bin/core/commands.dsc [hunk 5]
```diff
@@ -0,0 +1,503 @@
...
+        - killqs
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[null]>].if_null[true]>
+    script:
+        - if <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[server_or_command_block]>].if_null[true]>:
+            - if <player.exists>:
+                - if <context.args.size> == 0:
+                    - define warn <script[lib_config].parsed_key[color.warning]>
+                    - define soft <script[lib_config].parsed_key[color.soft_server_notice]>
+                    - if <util.queue.size> > 1:
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc]: I made a typo suggesting a change here and you committed the change before i could fix it, sorry :(

```suggestion
                    - if <util.queues.size> > 1:
```

File: utilities/bin/core/commands.dsc [hunk 6]
```diff
@@ -0,0 +1,503 @@
...
+                                - define always_valid true if:<[item].advanced_matches[*book]>
+                                - if <[item].material.name> == book:
+                                    - inventory adjust slot:<player.held_item_slot> material:enchanted_book
+                                - inventory adjust slot:<player.held_item_slot> enchantments:<[item_enchants].with[<[enchantment]>].as[<[level]>]>
+                                - inventory adjust slot:<player.held_item_slot> lore:<[item].lore.if_null[<list>].include[<gray><[enchantment].full_name[<[level]>]>]> if:<[enchantment].key.before[:].equals[minecraft].not>
+                                - narrate "<[colors].get[success]>You successfully enchanted a <[soft]><[item].material.translated_name><[colors].get[success]> with <[colors].get[important]><[enchantment].full_name[<[level]>]>"
+                                - foreach <[item_enchants]>:
+                                    - define ench <enchantment[<[key]>]>
+                                    - define conflicts:->:<[ench].full_name> if:<[enchantment].is_compatible[<[ench]>].not>
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc]: `enchantmenttag.full_name[<level>]`  requires level input, even if it's just the minimum level
```suggestion
                                    - define conflicts:->:<[ench].full_name[1]> if:<[enchantment].is_compatible[<[ench]>].not>
```

File: utilities/bin/core/commands.dsc [hunk 7]
```diff
@@ -0,0 +1,503 @@
+##IgnoreWarning command_script_usage
+##IgnoreWarning tag_trace_failure
+##IgnoreWarning bad_tag_part
```
Comment (BehrRiley) [inline utilities/bin/core/commands.dsc]: Both of these may be removed if the below tags are resolved
```suggestion
```

Comment (jumpsplat120) [reply utilities/bin/core/commands.dsc]: Already fixed.

File: utilities/bin/core/commands.dsc (hunk 6 above)
Comment (jumpsplat120) [reply utilities/bin/core/commands.dsc]: Already fixed.

File: utilities/bin/core/commands.dsc (hunk 5 above)
Comment (jumpsplat120) [reply utilities/bin/core/commands.dsc]: I saw lol. Already fixed.

**Review by BehrRiley — APPROVED** [review]
Fixed deprecated tags and removed warning overrides

**Pull request merged**